fhservices.cfg.j2

```diff
@@ -139,15 +144,24 @@ define service {
 }
 
 define service {
-       service_description Database::MongoDB::Health
-       check_command check_mongodb!mongodb,mongodb-service
+       service_description mongodb:health
+       check_command check_mongodb!mongodb,mongodb-service,mongodb-1
```

Comment:
Yes, this will fix RHMAP-9900.  Its the 3rd permutation of naming we have between mbaas and core.  I found it only when I used the latest core template again so I just fixed it while I was in there.